src/main/java/ca/corefacility/bioinformatics/irida/ria/web/announcements/AnnouncementsController.java

```diff
@@ -135,41 +134,6 @@ public String getControlCentreAdminPage(final Model model) {
         return ANNOUNCEMENT_ADMIN;
     }
 
```

Comment:
I see that the create and delete were removed from this controller and moved to the ajax controller. There remains however the update and delete methods. Are they used elsewhere or should they be removed as well?

Reply:
There will be a follow up merge request that handles those.  There is a separate page if you click on the message in the table that takes you to some detailed information (like who has read it, editing, and other stuff). I have a branch done with that in it, but it uses ant v4.

Reply:
Sounds good!

Reply:
Sounds good!